sem-service

```diff
@@ -334,6 +334,31 @@ service::err() {
   exit 1
 }
 
+################################################################################
+# Helper function to show docker start duration
+# Globals:
+#   DATE_FORMAT
+# Arguments:
+#   message
+# Returns:
+#   0
+################################################################################
+service::duration() {
+  local cmd="$@"
+  local start
+  local end
+  local duration
+
+  start=$(date +%s%3N)
+  $cmd
+  end=$(date +%s%3N)
+  duration=$(( $end - $start ))
+
+  echo "docker_runtime: ${duration}|ms" >> stats.txt
```

Comment:
If I change directories in my commands, will this also change the location of the file?

Example:

```
cd /tmp
sem-service postgres
```

will the output be in `/tmp/stats.txt`?

Reply:
I would suggest using an absolute path for this file, it is easier to document, to reason about, and minimizes ambiguity.

Reply:
@shiroyasha Good call, I've changed that to /tmp/stats.txt